pom.xml

```diff
@@ -48,7 +48,7 @@
     <argLine>-Dfile.encoding=${project.build.sourceEncoding}</argLine>
     <jenkins.version>2.60.3</jenkins.version>
     <java.level>8</java.level>
-    <jgit.version>5.0.2.201807311906-r</jgit.version>
+    <jgit.version>5.1.0.201808281540-m3</jgit.version>
```

Comment:
Are we fine with only a jgit milestone release?

Reply:
Not for a merge to the master branch.  I wanted to confirm that the fix in JGit really fixed the problem I had seen.  It did that.  

This will request will wait until the release of either JGit 5.1 or JGit 5.0.3.

Reply:
JGit v5.0.3.201809091024-r got released.